Student/FindStudentViewController.swift

```diff
@@ -83,33 +74,50 @@ class findStudentViewController: UIViewController,UITextFieldDelegate {
                 foundStudentPhoneNo.text=student.phoneNo
                 
             }
+            
+        }*/
+        let tempStudent = collegeDb.searchStudentWithRollNo(rollNo: Int(rollID.text!)! )
```

Comment:
don't use force unwrapping of optionals
function already has a parameter rollNo,, no need to add it in function
function name can be searchStudent(for rollNo: Int)